db/models.py

```diff
@@ -14,3 +14,35 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(to=Actor, related_name="actors")
```

Comment:
The `related_name` for the `actors` field should be more descriptive and not the same as the field name. Consider using a name like `movies_as_actor` to avoid confusion and potential conflicts.